book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
Please define a method for adding rentals in `Book` class. You may follow [this](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) link tp learn more about it.

Reply:
@SajjadAhmad14  no you mustn't there is a many-to-many relationship between the Person class and Book class so to associate them we defined an intermediate class Rental class so all methods to relate Person and Book if you read my test script and run it you will realize when you call rental class the rentals array inside either Book or Person objects updated automatically